woof-code/rootfs-skeleton/usr/sbin/dotpup

```diff
@@ -51,12 +51,12 @@ BOOTFLASHBUTTON=$(install_button \
 	"$(gettext "Quickly create a bootable usb flash drive")" \
 	'bootflash &' flashcard_usb.svg )
 
-if [ -d /usr/share/g4dos -o -f /usr/local/frugalpup/grub4dos.tar.xz -o -f /usr/lib/grub4dos/grldr ] ; then
+#if [ -d /usr/share/g4dos -o -f /usr/local/frugalpup/grub4dos.tar.xz -o -f /usr/lib/grub4dos/grldr ] ; then
```

Comment:
Why is it commented out? dpup doesn't have grub4dos installed and installation **will** fail if you blindly assume it exists.